build.sbt

```diff
@@ -3518,6 +3519,25 @@ lazy val `std-database` = project
       "org.xerial"           % "sqlite-jdbc"             % sqliteVersion,
       "org.postgresql"       % "postgresql"              % "42.4.0"
     ),
+    modulePath := {
+      val externalModIds = libraryDependencies.value ++ Seq(
```

Comment:
@Akirathan, I have no idea what this means or why such a complex setup for `modulePath` is necessary in the case of `std-database` project.

Reply:
This is a current technical limitation. Improvement tracked in https://github.com/enso-org/enso/issues/10726.